src/Sprites/IndexedBmpWriter.cpp

```diff
@@ -24,9 +24,9 @@ void IndexedBmpWriter::WriteScanLineIncluded(std::string filename, uint16_t bitC
 void IndexedBmpWriter::Write(std::string filename, uint16_t bitCount, int32_t width, int32_t height, const std::vector<Color>& palette, const std::vector<uint8_t>& pixels)
 {
 	ImageHeader::CheckIndexedBitCount(bitCount);
-	CheckPaletteCount(bitCount, palette.size());
-	CheckPixelCount(bitCount, std::abs(height) * width, pixels.size());
-	CheckPixelIndices(bitCount, palette.size(), pixels);
+	VerifyPaletteCount(bitCount, palette.size());
+	VerifyPixelCount(bitCount, std::abs(height) * width, pixels.size());
+	VerifyPixelIndices(bitCount, palette.size(), pixels);
```

Comment:
I ran with the function names. I think I modified one or two a bit from the suggestion. The original names were pretty poor on my part.

For paletteCount, I'm trying to convey the idea of the number of items in the container as opposed to the memory size of the palette container. After you mentioned it, I can see how this could be confusing. I'm worried paletteSize would give the impression of something like sizeof as opposed to range. 

What do you think about paletteRange???

Reply:
Hmm, maybe `paletteEntryCount`?

Reply:
Works for me, I'll push PaletteEntryCount